kythe/cxx/indexer/cxx/KytheIndexerMain.cc

```diff
@@ -57,6 +57,9 @@ ABSL_FLAG(bool, use_compilation_corpus_as_default, false,
           "Use the CompilationUnit VName corpus as the default.");
 ABSL_FLAG(bool, experimental_record_dataflow_edges, false,
           "Emit experimental dataflow edges.");
+ABSL_FLAG(std::string, template_instance_exclude_path_pattern, "",
+          "If nonempty, a regex that matches files to be excluded from "
+          "template instance indexing.");
```

Comment:
Per my other comment, I would convert this to a RE2 as eagerly as possible.  Ideally at argument-parsing time:
```
struct RE2Flag {
  std::unique_ptr<re2::RE2> value;
};

bool AbslParseFlag(absl::string_view text, RE2Flag* value, std::string* error) {
  re2::RE2::Options options;
  options.set_never_capture(true);
  *value = {absl::make_unique<re2::RE2>({text.data(), text.size()}, options)};
  if (!value->value->ok()) {
    *error = value->value->error_text();
  }
  return value->value.ok();
}
std::string AbslUnparseFlag(const RE2Flag& value) {
  return value.value->pattern();
}
```